src/microbe_stage/FloatingChunk.cs

```diff
@@ -125,6 +130,7 @@ public class FloatingChunk : RigidBody, ISpawned, ISaveLoadedTracked
         Size = chunkType.Size;
         Damages = chunkType.Damages;
         DeleteOnTouch = chunkType.DeleteOnTouch;
+        DamageType = string.IsNullOrEmpty(chunkType.DamageType) ? "chunk" : chunkType.DamageType;
```

Comment:
Doesn't this need handling in `CreateChunkConfigurationFromThis` as well? That seems like a bug just waiting to strike to me.

Reply:
Done, forgot about that.